### DISCUSSION TITLE
Isolated Network Issue

### DISCUSSION THREAD

**Discussion by kadirozdinc** [sandeepmistry/arduino-LoRa#440] [General]
Hi sandeep,

I am planning to create some lora networks which are isolated from each other, I want for these different networks not to affect each other. There is an examle of syncword but this technique is limited with 255,  I need more of them. How should I need to set up these isolated network.

(For examle, One lora network includes 1 sender and 4 receiver as duplex mode.)

**Comment by IoTThinks** [comment]
Sync Word won't guarantee 100% of isolation. There is still interference with Sync Word.
You can use different frequencies 920, 921, 922... => Zero interference.

Why do you need more than 255 isolated networks?

**Reply by kadirozdinc** [reply]
I have been currently designing a product that includes lora based communication and finally will send them to my customers. In case my customers are close to each other, these networks might mix up with each other. to prevent this, I want those networks to be isolated from each other. (lets say there are 300 customers which are close to each other)

Actually problem arise when I set my radio parameters for longest range, time on air is around 2 sec. Time on air also affects receiver side. Receiver is blocked for 2 sec while receiving data from sender. Datas coming from another networks can block the receiver  and then receiver cannot get data sent by corresponding sender. what can I do to overcome this ?

I use 433 mhz version named Lora Ra02 module. What is the usable range of this module ? For example I set 445 mhz for one of them and I set 415 mhz for another lora. In this setup, Modem's can interfere with each other.

**Reply by kadirozdinc** [reply]
Altough I use 433mhz for first lora and 434mhz for another lora, they interfere with each other. How is it possible ?

**Reply by IoTThinks** [reply]
Likely you are doing something not right in your code.
Or misunderstand something.

**Reply by kadirozdinc** [reply]
I just edited a little bit the original code:

Sender:

`#include <SPI.h>
#include <LoRa.h>

// LORA PINs
#define ss 5
#define rst 14
#define dio0 2

int counter = 0;

void setup() {
  Serial.begin(9600);
  while (!Serial);

  Serial.println("LoRa Sender");
  LoRa.setPins(ss, rst, dio0);
  while (!LoRa.begin(433E6)) {
    Serial.println(".");
    delay(500);
  }

  LoRa.setSpreadingFactor(12);
  LoRa.setSignalBandwidth(62.5E3);
  LoRa.setCodingRate4(8);
  LoRa.setTxPower(17);
  LoRa.enableCrc();
}

void loop() {
  Serial.print("Sending packet: ");
  Serial.println(counter);

  // send packet
  LoRa.beginPacket();
  LoRa.print("hello ");
  LoRa.print(counter);
  LoRa.endPacket();

  counter++;

  delay(5000);
}`

Receiver

`#include <SPI.h>
#include <LoRa.h>

// LORA PINs
#define ss 5
#define rst 14
#define dio0 2

void setup() {
  Serial.begin(9600);
  while (!Serial);

  Serial.println("LoRa Receiver");
  
  LoRa.setPins(ss, rst, dio0);
  while (!LoRa.begin(434E6)) {
    Serial.println(".");
    delay(500);
  }

  LoRa.setSpreadingFactor(12);
  LoRa.setSignalBandwidth(62.5E3);
  LoRa.setCodingRate4(8);
  LoRa.setTxPower(17);
  LoRa.enableCrc();
}

void loop() {
  // try to parse packet
  int packetSize = LoRa.parsePacket();
  if (packetSize) {
    // received a packet
    Serial.print("Received packet '");

    // read packet
    while (LoRa.available()) {
      Serial.print((char)LoRa.read());
    }

    // print RSSI of packet
    Serial.print("' with RSSI ");
    Serial.println(LoRa.packetRssi());
  }
}`

**Reply by IoTThinks** [reply]
How do you know they interfere with each others?

**Reply by kadirozdinc** [reply]
At Receiver side, They can stil receive the messages that sender sends. What I mean by interfering, Receiver can get the messages sender transmits altough they are on different frequency band. I want receiver not to hear comming messages from different frequencies I just want isoation between different networks

**Reply by IoTThinks** [reply]
Show us the photo of your kits?
Normally, even 433E6 and 433.2E6 won't interfere with each other.

**Reply by kadirozdinc** [reply]
[https://calcuttaelectronics.com/wp-content/uploads/2018/11/ora-02-breakout-board_1.jpg](url)

I use these modules, 433 mhz sx1278 based. What is the usable frequency range of this one ? And how much frequency do I need to leave in between not to interfere ?

**Reply by IoTThinks** [reply]
Usable frequency: From 410-525Mhz.
Each channel should be at least 1/2 * Signal BW from each other.

**Reply by IoTThinks** [reply]
Check your antenna if it is for 433Mhz.

**Reply by kadirozdinc** [reply]
I use cheap FPC antenna from china right now. Since I dont have vna device, I cannot check if its in the right range. Lets say my antenna is out of range, does it cause interference ?

LoRa covers very wide range of area and many users might prefer to use LoRa. I am wondering how these users can isolate their networks to prevent interference ?

**Reply by IoTThinks** [reply]
Something is really incorrect at your side.
Double check everything again.

**Reply by kadirozdinc** [reply]
I change bandwidth to 250khz after 125khz problem solved for now. To be able to isolate networks, can preambleLength parameter be used ? What does preambleLength parameter do ?

**Reply by IoTThinks** [reply]
Preample is to know where is the beginning of an LoRa packet.

**Reply by kadirozdinc** [reply]
Yes it is ,why do we need to use it ? I have found an explanation about it

Preamble: used to keep the receiver synchronized with the incoming data stream. The default is 12 symbol lengths, and 8 symbol lengths are used in LoRaWAN. The preamble length is a variable that can be set by programming, so the length of the preamble can be extended. The length of the preamble of the receiver should be the same as that of the transmitter. If the preamble length is unknown or may change, the receiver’s preamble length should be set to the maximum value. **Address filtering can be performed by setting the preamble value to achieve packet communication.**

If so, It seems I can use it  for address filtering
